shared/src/business/useCases/filePetitionInteractor.test.js

```diff
@@ -1,8 +1,17 @@
+const {
+  uploadDocumentAndMakeSafeInteractor,
+} = require('./uploadDocumentAndMakeSafeInteractor');
 const { applicationContext } = require('../test/createTestApplicationContext');
 const { filePetitionInteractor } = require('./filePetitionInteractor');
 const { ROLES } = require('../entities/EntityConstants');
 
 beforeAll(() => {
+  applicationContext
+    .getUseCases()
+    .uploadDocumentAndMakeSafeInteractor.mockImplementation(
```

Comment:
I see this is done in a couple of tests, could this be moved to `createTestApplicationContext`?